src/pages/home/ReportScreen.tsx

```diff
@@ -540,6 +539,10 @@ function ReportScreen({
                 Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(prevReport.parentReportID));
                 return;
             }
+            // If the report isn't focused, navigation to Concierge Chat should be avoided.
+            if (!isFocused) {
+                return;
+            }
```

Comment:
I think the return early should be in the first line in the block of code, there is no need for any navigation call if navigation is not focused. I think this is more clean. What do you think?